.docker/readme.md

```diff
@@ -8,8 +8,6 @@ Follow these steps to setup a local WordCamp.org environment using [Docker](http
     ```bash
     git clone git@github.com:WordPress/wordcamp.org.git wordcamp.test
     cd wordcamp.test
-	rm -rf .git/hooks
-	ln -s .githooks .git/hooks
     ```
```

Comment:
Why is this part removed?

Reply:
Sorry, I should have made this change as a separate commit and explained it.

While I think it's a good idea to use hooks to ensure better code quality in PRs we get from contributors, the hooks aren't functional out-of-the-box. The scripts make assumptions that tools like phpcs are installed, and about where they are installed, which may not work for everyone (they didn't work for me). So I think the scripts need some updates (using `$(which phpcs)` for example), and we should probably add the installation and configuration of the necessary tool dependencies as a separate step in the Initial Setup process.

In the mean time, I removed them because in the process of doing this refactor, I was setting up the Docker environment from scratch and going through these steps, and i realized that those lines might be a stumbling point for other folks who are trying to get the environment up and running.

Reply:
> we should probably add the installation and configuration of the necessary tool dependencies as a separate step in the Initial Setup process.

Or, this could potentially be automated in composer and/or the Docker script.

Reply:
🤔 those are good considerations. I'm a bit leery of removing them entirely, though. 

I wonder if there's a middle ground, like marking them as optional steps, and mentioning that  the documentation is still a work in progress, so that people feel free to skip them, or give up if they can't get work them working quickly?

Reply:
Could we use something like [husky](https://www.npmjs.com/package/husky) for this? I know we don't currently use `npm` in the root of the project, but this might be a good reason?

Also, since phpcs has been in composer since #185, we can say `composer run phpcs …` to run the vendor script, instead of relying on a global install.

Reply:
> husky

I looked at the docs, but they don't really explain what it does. What about it do you find useful?

> composer run phpcs

Great point :)

Reply:
> I looked at the docs, but they don't really explain what it does.

It just handles setting up git hooks automatically so we don't need to tell people to sync githooks like this. We can then use custom scripts or npm commands, the hooks work just like the `scripts` part of package.json.

Reply:
Added the git hook commands back in as an optional step in f343eb1e

I think it's worth looking into automating the git hook stuff, either with husky or something else, but I'd rather open an issue to discuss it along with other improvements to the precommit hook scripts, and not deal with it in this PR.

Reply:
Issue: https://github.com/WordPress/wordcamp.org/issues/228